ocaml/xapi-consts/api_errors.ml

```diff
@@ -248,6 +248,7 @@ let vdi_in_use = "VDI_IN_USE"
 let vdi_is_sharable = "VDI_IS_SHARABLE"
 let vdi_readonly = "VDI_READONLY"
 let vdi_too_small = "VDI_TOO_SMALL"
+let vdi_too_large = "VDI_TOO_LARGE"
```

Comment:
We also need to defined this error in a second file, like in the original PR: https://github.com/xapi-project/xen-api/pull/3788/files#diff-394b7fa65daaf936d3837fae220fb1b9R709